clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/model/OpenstackLoadBalancer.groovy

```diff
@@ -28,7 +28,7 @@ import org.openstack4j.model.network.ext.HealthMonitor
 import org.openstack4j.model.network.ext.LbPool
 
 @Canonical
-@JsonIgnoreProperties(['nameRegex','namePattern','descriptionRegex','descriptionPattern'])
+@JsonIgnoreProperties(['nameRegex','namePattern','portRegex','portPattern','createdRegex','createdPattern'])
```

Comment:
I believe nameRegex and Pattern are no longer being used ... can you please remove or I can follow up in another PR.